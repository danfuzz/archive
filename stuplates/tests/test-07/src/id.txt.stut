== true
${ (:id: null == null) }
${ (:id: true == true) }
${ (:id: false == false) }
${ (:id: "x" == "x") }
${ def a := 1; (:id: a == a) }

== false
${ (:id: null == true) }
${ (:id: true == false) }
${ (:id: "x" == (:default: "x" + "")) }

!= true
${ (:id: null != true) }
${ (:id: true != false) }
${ (:id: "x" != (:default: "x" + "")) }

!= false
${ (:id: null != null) }
${ (:id: true != true) }
${ (:id: false != false) }
${ (:id: "x" != "x") }
${ (:id: a != a) }

+
expect nothing: ${ +null }
${ +"blort" } blort
${ +10 } 10
${ +10.5 } 10.5
