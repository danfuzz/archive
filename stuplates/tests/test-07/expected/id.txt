== true
true
true
true
true
true

== false
false
false
false

!= true
true
true
true

!= false
false
false
false
false
false

+
expect nothing: 
blort blort
10 10
10.5 10.5
